configuration/README.md

```diff
@@ -37,35 +49,50 @@ FieldTags:
   Val: source
 ```
 
-Sinks and sanitizers are identified via regexp according to package, method, and (optional) receiver name.
+Sinks and sanitizers are identified by package, method, and (optional) receiver name.
+Again, these may be specified by either a provided string literal or regexp.
 
 ```yaml
 Sinks:
-- PackageRE: <package path regexp>
+- Package:  "literal/package/path" 
   ReceiverRE: <type name regexp>
   MethodRE: <method name regexp>
 Sanitizers:
 - PackageRE: <package path regexp>
   ReceiverRE: <type name regexp>
-  MethodRE: <method name regexp>
+  Method: "mySanitizer"
 ```
 
 Taint propagation is performed automatically and does not need to be explicitly configured.
 
-For matchers that accept a `ReceiverRE` regexp matcher, an unspecified string will match any (or no) receiver.
-To match only methods without any receiver (i.e., a top-level function), use the matcher `^$` to match an empty-string receiver name.
+### Important Note
+
+Configuration of the above matchers does not require listing of all arguments.
+E.g., the following is a valid source configuration:
+```yaml
+Sources:
+- Package: "literal/package/path"
+  Type: "MySourceType"
+```
+
+Neither `Field` nor `FieldRE` have been provided.
+In this case, we match all fields of `MySourceType`, assuming a wildcard matcher for `FieldRE`.
```

Comment:
Nit: I think the part after the comma is unnecessary and potentially confusing. It might help to expand a little, e.g. : `If you want to be more explicit, you may use an empty regex such as FieldRE: "". An empty regex will match anything.`